### DISCUSSION TITLE
my github page can't be reached

### DISCUSSION THREAD

**Discussion by sharkxhunter** [community/community#50899] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/126171640/227471635-b4516365-83a1-48e7-bfa2-e5e30ed67c40.png)

devking877.github.io
This is my github page and can't be reached from anywhere.
Please help me.

**Comment by neutrinomuon** [comment]
Hi,

I was able to publish my website at 'https://username.github.com' by creating a repository with the name 'username.github.io'. In my case, I used 'neutrinomuon.github.io' as the repository name. If you simply enable GitHub Pages for a repository, it will only be published at https://username.github.io/repo.

Please, let me know if it works... I hope this helps!

**Comment by sharkxhunter** [comment]
Thanks for  your advice. I tried your advice, but still same.
When not deployed, the other user's github pages show 404 error, but my
github page shows an error 'can't be reached'.
Thanks again.

**Reply by neutrinomuon** [reply]
Hum, you're welcome, but when I try to reach https://devking877.github.io/ it shows a 404 error in my browser...

**Comment by sharkxhunter** [comment]
Really? But I can't see 404 error. Could you try this link
https://devking877.github.io/portfolio/, please? 'portfolio' is my
portfolio repository and already deployed, but I can't see. Could you check
it and send me screenshot please?

**Reply by neutrinomuon** [reply]
It shows this:

<img width="1440" alt="ScreenShot 2023-03-24, às 15 25 22" src="https://user-images.githubusercontent.com/8226984/227569440-f3f6f1ef-54de-424d-b5fc-a13bbc2294ff.png">

I'm using google chrome

**Comment by sharkxhunter** [comment]
Thanks, it works. But I still cannot access to that page. Could you try one
more please?
https://devking877.github.io   and please send me a screenshot.

**Reply by neutrinomuon** [reply]
This is strange! I have access find the screenshot below:

<img width="1440" alt="Screenshot 2023-03-24, às 18 34 50" src="https://user-images.githubusercontent.com/8226984/227611408-e5fd958f-5e99-473f-8bb7-4327d56c3648.png">

**Comment by sharkxhunter** [comment]
Got it. Thanks very much.

**Comment by anshtiwatne** [comment]
After going through your github actions it seems like the error is originating when pulling ghcr.io, try reuploading your files to the repository
<img width="555" alt="image" src="https://user-images.githubusercontent.com/83647366/227699694-c1ab61e9-7cf3-4757-9866-9567a9f9bc47.png">
Here's a simillar problem I found on [stackoverflow](https://stackoverflow.com/questions/71997710/github-pages-error-docker-pull-failed-with-exit-code-1)

**Comment by sharkxhunter** [comment]
Thanks for your effort. Could you try again once more? It'll work, I guess.
It was really hard work. Thanks again.

**Comment by anshtiwatne** [comment]
Seems like your pages workflow is running error free now, your website looks to be rendering perfectly!
<img width="684" alt="image" src="https://user-images.githubusercontent.com/83647366/227709907-c1e053ed-e500-4f0d-affc-3f81af916c26.png">

<img width="777" alt="image" src="https://user-images.githubusercontent.com/83647366/227710089-43bd177a-7f4f-4486-8bc1-ddbc202e68d2.png">

**Comment by sharkxhunter** [comment]
Great, thanks!

**Reply by neutrinomuon** [reply, marked as answer]
You're welcome :)! I'm happy to know that it worked

**Comment by sharkxhunter** [comment]
Glad to hear it! You help was an important factor. Thanks a lot. I think we
can set up good partnership. I hope we'd discuss more in various aspects in
the future. Thanks again.

**Comment by batexbafika** [comment]
hi please can someone help check what the pobmen is? my page takes a long response time and will not display "cant reach this site is what i get . help https://batexbafika.github.io/internship-project-1/

**Reply by neutrinomuon** [reply]
Hi, I went in your page and everything is working well. Sometimes it takes a while (usually a few minutes) for githubpages to refresh. Are you still having problems? I've just accessed your page and it seems very well done :)!

**Reply by batexbafika** [reply]
wow  thats good news! please for image of the page please

**Reply by neutrinomuon** [reply]
I just noticed the message now. I attempted to revisit the page a few minutes ago, and the calculator looks different. I couldn't figure out how to use it this time, and its appearance has changed significantly since my last visit.

<img width="247" alt="Calculator" src="https://github.com/community/community/assets/8226984/99eab3f7-7e0a-46c2-a748-0144b19efec2">

**Reply by batexbafika** [reply]
yes sorry about that i just linked my **css** and **js** now try it again please.

**Reply by sharkxhunter** [reply]
![image](https://github.com/community/community/assets/126171640/eddbbccf-caaf-47d6-9802-a281dcae3929)
It seems working well.

**Reply by batexbafika** [reply]
THK U VMUCH

**Comment by 77v7777** [comment]
You have to write the name of git repo as your user name.

For example, if your github username is ABCDE, you should name the repo as ABCDE.

Then, you can see _https://ABCDE.github.io_

**Comment by Polybotics75** [comment]
Please guys i need help, i just realized that all my repository (github pages) are showing a "can not be reached" error

**Reply by neutrinomuon** [reply]
Hi, I've tried to access one f your repository https://github.com/Polybotics75/polyclock, but it says "The logs for this run have expired and are no longer available.". Can you try to make a new deployment so that we can see the error?

**Reply by Polybotics75** [reply]
Ok sir, i have deployed that particular "polyclock" repo

**Comment by Polybotics75** [comment]
Good day sir/ma,

Like you requested, i have redeployed that particular "polyclock" repo

**Reply by sharkxhunter** [reply]
It seems working now
